packages/internet-header/src/components/post-main-navigation/components/level-one-action.component.tsx

```diff
@@ -31,6 +32,9 @@ export const LevelOneAction = (props: {
       onClick={e => props.onClick(e)}
     >
       <span>{props.level.text}</span>
+      {props.level.isActiveOverride ? (
+        <span class="visually-hidden">{translate('Active navigation point')}</span>
```

Comment:
The text should probably be added to the active flyout item too: 
![image](https://github.com/swisspost/design-system/assets/33580481/6c0918cb-4219-4efb-a856-051525ff3b10)